src/tests/intg/test_kcm.py

```diff
@@ -355,9 +355,8 @@ def exercise_kswitch(testenv):
     assert len(cc_coll) == 2
     assert set(cc_coll['alice@KCMTEST']) == set(['krbtgt/KCMTEST@KCMTEST',
                                                  'host/somehostname@KCMTEST'])
-    assert set(cc_coll['bob@KCMTEST']) == set([
-                                    'krbtgt/KCMTEST@KCMTEST',
-                                    'host/differenthostname@KCMTEST'])
+    assert set(cc_coll['bob@KCMTEST']) == set(
+        ['krbtgt/KCMTEST@KCMTEST', 'host/differenthostname@KCMTEST'])
```

Comment:
Could you explain why it is not a problem with python-pep8-1.6.2-9?

Reply:
It depends whether E126 is defined to be ignored as default or not.